source/acquaman/REIXS/REIXSXESScanController.cpp

```diff
@@ -182,6 +182,16 @@ bool REIXSXESScanController::startImplementation() {
 	return true;
 }
 
+void REIXSXESScanController::pauseImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->pauseDwelling();
+}
+
+void REIXSXESScanController::resumeImplementation()
+{
+	REIXSBeamline::bl()->mcpDetector()->resumeDwelling();
+}
```

Comment:
What happens to the detector in cases where the Scan Controller is paused, and then cancelled or skipped? Will the veto remain active for the detector?